index.md

```diff
@@ -36,15 +36,21 @@ Did we mention that this place is above the arctic circle? Which is pretty cool.
     </div>
 </section>
 
-## History
+## Registration
 
-Since 2015 codefreeze has happened every year around January. 2018 we hade more than 25 participants plus their family and friends. 
+- Read and accept the [Code of Conduct](#code-of-conduct).
+- Book a room or cabin by email (see [Accommodation](#accommodation)), or ask for a roommate from our [Slack channel](#slack).
+- Make travel arrangements to Kiilopää (see [Directions](#directions)).
+- You may add yourself as a [participant](#participants) to the Codefreze website by creating a [pull request](https://github.com/rinkkasatiainen/codefreeze.fi#readme).
 
-To get an idea of what happened you can find the conference schedule from [2017](https://trello.com/b/zIYeI0me/codefreeze-2017) and [2018](https://trello.com/b/WtOxnkMU/codefreeze-2018) on Trello. Most *codefreeze* topics (green) have additional information in links in the cards.
+
+## Past Events
+
+Since 2015, Codefreeze has happened every year around January. In 2019 we had about 30 participants plus their family and friends.
```

Comment:
```suggestion
Since 2015, Codefreeze has happened every year around January. In 2019 we had about 35 participants plus their family and friends.
```